packages/taquito-ledger-signer/src/error.ts

```diff
@@ -1,3 +1,5 @@
+export { InvalidDerivationPathError } from '@taquito/core';
```

Comment:
i have done these to keep them exportable from the same file to make it exportable from the index of this package or the file that contains it

Reply:
to stop breaking changes of moving it to taquito core

Reply:
wouldn't it make more sense to keep InvalidDerivationPathError in this package instead of moving it up to`core`? Or was there a reason why it might need to be put up there?

Reply:
if it's not like a shared across packages thing, like say, `InvalidAddressError` or something like that. Might want to keep it contained in the package it's referenced at instead. But I might be missing context

Reply:
its used in both taquito signer and taquito ledger signer